client/src/components/Product-items/Products.js

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import { connect } from "react-redux";
+import { gql } from "apollo-boost";
+import { Query, Mutation } from "react-apollo";
+
+import { onAddedItem } from "../../redux/shopCart/cart-actions";
+
+import {
+  ProductContainer,
+  ProductCard,
+  MediaImage,
+  ProductFooterContainer,
+  ProductInfoContainer,
+  AddButton,
+  LoadingProduct
+} from "./products-styles.js";
+
+const LIKE_PRODUCT = gql`
+  mutation LikeProduct($userId: String!, $productId: String!) {
+    likeProduct(userId: $userId, productId: $productId) {
+      userId
+    }
+  }
+`;
+
+const GET_PRODUCTS = gql`
+  {
+    merchants {
+      guid
+      merchant
+      products {
+        id
+        name
+        price
+        description
+        color
+        size
+        image
+        quantity
+      }
+    }
+  }
+`;
+
+const withLike = Component => props => {
```

Comment:
when user likes a product, it will send information like userId and productId to the database